linkerd/proxy/http/src/h1.rs

```diff
@@ -19,23 +20,27 @@ pub fn normalize_our_view_of_uri<B>(req: &mut http::Request<B>) {
 
     // try to parse the Host header
     if let Some(auth) = authority_from_host(&req) {
+        trace!("using host header");
         set_authority(req.uri_mut(), auth);
         return;
     }
 
     // last resort is to use the so_original_dst
-    let orig_dst = req
+    if let Some(addr) = req
         .extensions()
-        .get::<Source>()
-        .and_then(|ctx| ctx.orig_dst_if_not_local());
-
-    if let Some(orig_dst) = orig_dst {
+        .get::<tls::accept::Meta>()
+        .map(|s| s.addrs.target_addr())
+    {
+        trace!(target.addr = %addr, "using target address");
```

Comment:
nit: should this say
```suggestion
        trace!(target.addr = %addr, "using original dst target address");
```
or something?

Reply:
I'm trying to set up most of the uses around accept::Meta/listen::Addrs to be agnostic about whether the orig dst is set (it's kind of an impl detail) to support serving local services from the proxy directly